Dockerfile

```diff
@@ -1,72 +1,22 @@
-# EXT_REGISTRY allows specifying a different registry that is acting as a pull-through cache for Docker Hub.
-# To provide EXT_REGISTRY at build time, supply it as a build argument
-#     `docker build . --build-arg EXT_REGISTRY=https://myregistry.com/`
-# Note that the value MUST contain a trailing slash
-# If not provided, images will be pulled from Docker Hub
-ARG EXT_REGISTRY=""
-ARG IMAGE_TAG="11.4"
+FROM python:3.10-slim-bullseye
 
-FROM ${EXT_REGISTRY}docker.io/library/debian:${IMAGE_TAG} as proj
-# Renew build args for the stage
-ARG EXT_REGISTRY IMAGE_TAG
+RUN apt-get update && apt-get -y upgrade
+RUN apt-get install -y wget git libopenblas-dev imagemagick g++ make imagemagick
```

Comment:
Should be able to remove imagemagick here.

Reply:
I figured that'd be best done by merging with main (now that PR #340 has been merged) for organizations sake - would it be more conventional to remove in this PR?

Reply:
In the interest of making this a minimal/functional Dockerfile, I think it is probably fine to remove it here (basically re-evaluating all the dependencies in this dockerfile independently of that other PR).  Maybe @jsolbrig has a different opinion?

Reply:
Fixed in most recent commit :)